tests/testthat/test-5transformCounts.R

```diff
@@ -70,6 +70,20 @@ test_that("transformAssay", {
                          log2(x+5)
                      }), check.attributes = FALSE)
         
+        ############################ CSS ######################################
+        # Apply CSS normalization using transformAssay
+        tmp <- mia::transformAssay(tse, method = "css")
+        ass <- assays(tmp)$css
+        
+        # Calculates CSS normalization. Should be equal.
+        mat <- as.matrix(assay(tse, "counts"))
+        
+        # Manually compute CSS normalization
```

Comment:
Should we also include comparison with an external package implementation? Or at least you could test that manually and include the code here even if it is commented out.

Reply:
so yeah my method and metagenomeSeqs give quite different results. This is their code - https://github.com/HCBravoLab/metagenomeSeq/blob/master/R/cumNorm.R - which obviously has quite a few more steps than my current implementation. Should I just add those steps into my implementation in a way that fits with mia?

Reply:
They seem to have fast variants, could you check if that's useful?

https://github.com/HCBravoLab/metagenomeSeq/blob/master/R/cumNormStatFast.R

Reply:
yeah I was gonna ask you - I'm not too sure - I know that for alot of datasets the end value usually ends up being 0.5 anyway, so not sure how needed it is?

Reply:
Would it harm to allow users choose it if most of the time they do not have to worry about it when the default is good?

Reply:
No it wouldn't harm, it just makes the method slightly more complex, but I can add it - should I add it?

Reply:
Yes, this would be the standard way to do. We like to support user freedom even if the defaults are good.